applications/ULFapplication/custom_processes/find_nodal_h_process.h

```diff
@@ -182,7 +182,7 @@ class FindNodalHProcess
 
         //calculate average h
         if(h_nodes == 0)
-            KRATOS_THROW_ERROR(std::logic_error,"no node has neighbours!!!!","");
+            KRATOS_ERROR(std::logic_error,"no node has neighbours!!!!","");
```

Comment:
There's a common find_nodal_h_process in Kratos Core. It would be nice if you could use that one to avoid code duplication.